### PR TITLE
bchunk: add page

### PR DESCRIPTION
**Pull request by 4G3NT**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
bchunk

### CODE REVIEW

**Comment by 4G3NT** [comment]
The command is fairly small so I can't really put 8 examples.

**Comment by 4G3NT** [comment]
It appears that you don't need to add .[EXTENSION] in front of the output as bchunk will do that for you.

File: pages/linux/bchunk.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# bchunk
+
+> Command-line tool for converting .bin files to .iso files.
```
Comment (acuteenvy) [inline pages/linux/bchunk.md:3]:
```suggestion
> Convert `.bin` files to `.iso` files.
```

File: pages/linux/bchunk.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <http://he.fi/bchunk>.
+
+- Convert binary CD into a standard iso9960 image file:
+
+`bchunk {{image.bin}} {{image.cue}} {{output}}`
```
Comment (acuteenvy) [inline pages/linux/bchunk.md:8]:
```suggestion
`bchunk {{path/to/image.bin}} {{path/to/image.cue}} {{path/to/output}}`
```

File: pages/linux/bchunk.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Convert with verbose mode:
+
+`bchunk -v {{image.bin}} {{image.cue}} {{output}}`
```
Comment (acuteenvy) [inline pages/linux/bchunk.md:12]:
```suggestion
`bchunk -v {{path/to/image.bin}} {{path/to/image.cue}} {{path/to/output}}`
```

File: pages/linux/bchunk.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Output audio files in WAV format:
+
+`bchunk -w {{file.bin}} {{image.cue}} {{output}}`
```
Comment (acuteenvy) [inline pages/linux/bchunk.md:16]:
```suggestion
`bchunk -w {{path/to/image.bin}} {{path/to/image.cue}} {{path/to/output}}`
```

**Comment by 4G3NT** [comment]
👍

File: pages/linux/bchunk.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# bchunk
+
+> Convert `.bin` files to `.iso` files.
```
Comment (kbdharun) [inline pages/linux/bchunk.md:3]:
```suggestion
> Convert CD images to a set of `.iso` and `.cdr` tracks.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by 4G3NT** [comment]
More descriptive suggestion 👍

**Review by kbdharun — APPROVED** [review]

**Comment by 4G3NT** [comment]
Had to force push & amend to update the commit message.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**